spec/features/admin/bulk_operations/update_and_verify_trns_spec.rb

```diff
@@ -0,0 +1,108 @@
+require "rails_helper"
+
+RSpec.feature "update and verify TRNs", :rack_test_driver, type: :feature do
```

Comment:
There's not a request spec to check that an admin account is required to access this. Can either one be added or can this feature spec be extended to do authentication checks, thanks